priv/samples/tflite.livemd

```diff
@@ -33,36 +32,7 @@ reference](https://hexdocs.pm/tflite_elixir/api-reference.html).
 ```elixir
 defmodule Utils do
   def download!(source_url, req_options \\ []) do
-    Req.get!(source_url, [finch_request: &finch_request/4] ++ req_options).body
-  end
-
-  defp finch_request(req_request, finch_request, finch_name, finch_options) do
-    acc = Req.Response.new()
-
-    case Finch.stream(finch_request, finch_name, acc, &handle_finch_stream/2, finch_options) do
-      {:ok, response} -> {req_request, response}
-      {:error, exception} -> {req_request, exception}
-    end
-  end
-
-  defp handle_finch_stream({:status, status}, response), do: %{response | status: status}
-
-  defp handle_finch_stream({:headers, headers}, response) do
-    req_headers = headers |> Enum.map(fn {k, v} -> {k, List.wrap(v)} end) |> Map.new()
-    total_size = req_headers |> Map.fetch!("content-length") |> List.first() |> String.to_integer
-
-    response
-    |> Map.put(:headers, req_headers)
-    |> Map.put(:private, %{total_size: total_size, downloaded_size: 0})
-  end
-
-  defp handle_finch_stream({:data, data}, response) do
-    new_downloaded_size = response.private.downloaded_size + byte_size(data)
-    ProgressBar.render(new_downloaded_size, response.private.total_size, suffix: :bytes)
-
-    response
-    |> Map.update!(:body, &(&1 <> data))
-    |> Map.update!(:private, &%{&1 | downloaded_size: new_downloaded_size})
+    Req.get!(source_url, req_options).body
```

Comment:
This might not need to be a module anymore. But it is fine to leave it as this for now :)